src/hotel-management-app/src/app/customer/login/login.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { CustomerService } from '../../services/customer.service';
+
+@Component({
+  selector: 'app-customer-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.css']
+})
+export class LoginComponent {
+  loginForm: FormGroup;
+  errorMessage: string | null = null;
+
+  constructor(private fb: FormBuilder, private customerService: CustomerService, private router: Router) {
+    this.loginForm = this.fb.group({
```

Comment:
Could you explore if we could typecast FormBuilder Similar to how we typecast FormGroup